library/std/src/lib.rs

```diff
@@ -12,6 +12,9 @@
 // re-export all std symbols
 pub use std::*;
 
+// Override process calls with stubs.
+pub mod process;
```

Comment:
I vaguely recall us having issues with overriding things here that weren't macros. Was that resolved somehow?

Reply:
The one I recall failing to override  was `std::mem::swap`, which turned out was because we have a hook for it: https://github.com/model-checking/kani/blob/e20874a616c0e2726726d4e23143ce77077ea7e2/src/kani-compiler/src/codegen_cprover_gotoc/overrides/hooks.rs#L390